clients/admin-ui/src/features/data-discovery-and-detection/action-center/fields/page.tsx

```diff
@@ -252,68 +278,82 @@ const ActionCenterFields: NextPage = () => {
                   </Button>
                 </Dropdown>
                 <Dropdown
+                  onOpenChange={onActionDropdownOpenChange}
                   menu={{
                     items: [
                       ...DROPDOWN_ACTIONS.map((actionType) => ({
                         key: actionType,
                         label: FIELD_ACTION_LABEL[actionType],
-                        disabled: selectAll
-                          ? false
+                        disabled: isFetchingAllowedActions
+                          ? true
                           : !availableActions?.includes(actionType),
```

Comment:
This can be simplified
```suggestion
                        disabled: isFetchingAllowedActions || !availableActions?.includes(actionType),
```